tutorials/source-modeling/plot_dipole_fit.py

```diff
@@ -101,3 +101,10 @@
 dip_fixed = mne.fit_dipole(evoked_full, fname_cov, fname_bem, fname_trans,
                            pos=dip.pos[best_idx], ori=dip.ori[best_idx])[0]
 dip_fixed.plot(time_unit='s')
+
+##############################################################################
+# References
+# ----------
+# .. [1] Sarvas, J. (1987). Basic mathematical and electromagnetic concepts of
```

Comment:
Probably worth a `[1]_` actually referencing it in the text somewhere above

Reply:
Done